decidim-core/app/packs/stylesheets/decidim/_layout.scss

```
@@ -31,7 +31,7 @@
 }
 
 .layout-2col {
-  @apply md:grid grid-cols-12 container grow;
+  @apply md:grid grid-cols-12 container grow auto-rows-fr;
```

It looks like the grid requires the same min-height as its parent has

```suggestion
  @apply md:grid grid-cols-12 container grow min-h-[60vh];
```

Tested in browserstack, try in local and let me know if it works